misc/systemd/cf-serverd.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=CFEngine Enterprise file server
+After=syslog.target
+After=network.target
+ConditionPathExists=@bindir@/cf-serverd
+PartOf=cfengine3.service
+
+[Service]
+Type=simple
+ExecStart=@bindir@/cf-serverd --no-fork --verbose
```

Comment:
I think I've said before that I don't agree the daemons should be launched in verbose mode, especially not `cf-serverd` which can generate massive amounts of logs that way. But it's up to you and @jimis, ultimately.